erpnext/projects/doctype/task/task.json

```diff
@@ -116,11 +116,12 @@
    "fieldtype": "Select",
    "in_list_view": 1,
    "in_standard_filter": 1,
-   "label": "Status",
+   "label": "Default Status",
```

Comment:
leave the label as Status, don't change it, instead add a description that the status of the default project will be set.